src/inmanta/data/model.py

```diff
@@ -157,6 +157,22 @@ class CompileReport(CompileRunBase):
     version: Optional[int]
 
 
+class Report(BaseModel):
```

Comment:
I wound make the name more specific than just `Report`. It's very likely that we will add other dto's in the future that will represent some kind of report. That would be confusing.